### PR TITLE
NICTIZ-22485-22486-22483: CiO 2.0.0-beta.3 (HL7v3 instances)

### PR DESCRIPTION
**Pull request by LuudSlagter**

### CODE REVIEW

File: ada_2_hl7/cio/2.0.0/sturen_antwoord_voorstel_contra_indicatie/hl7_instance/cio-savc-tst-1.2-AVCI1.xml
Comment (alayabenbelkacem) [inline ada_2_hl7/cio/2.0.0/sturen_antwoord_voorstel_contra_indicatie/hl7_instance/cio-savc-tst-1.2-AVCI1.xml:1]: no Identificationnumber from RegInfo in this instance

File: ada_2_hl7/cio/2.0.0/sturen_antwoord_voorstel_contra_indicatie/hl7_instance/cio-savc-tst-1.4-AVCI2.xml
Comment (alayabenbelkacem) [inline ada_2_hl7/cio/2.0.0/sturen_antwoord_voorstel_contra_indicatie/hl7_instance/cio-savc-tst-1.4-AVCI2.xml:1]: no Identificationnumber from RegInfo in this instance

Comment (LuudSlagter) [reply ada_2_hl7/cio/2.0.0/sturen_antwoord_voorstel_contra_indicatie/hl7_instance/cio-savc-tst-1.4-AVCI2.xml:1]: Solved!

File: ada_2_hl7/cio/2.0.0/sturen_voorstel_contra_indicatie/hl7_instance/cio-svci-tst-1.3-VCI2.xml
[the remark this answers is not in the document]
Comment (LuudSlagter) [reply ada_2_hl7/cio/2.0.0/sturen_voorstel_contra_indicatie/hl7_instance/cio-svci-tst-1.3-VCI2.xml:1]: See comment above.

**Review by njnikodym — APPROVED** [review]

**Pull request merged**